app/views/application/index.html.erb

```diff
@@ -1,5 +1,6 @@
-<section class="front-section">
+<section class="front-section banner" style="background-image: url('<%= image_path('hpi_main_building_keyvisual_1020x420.jpg') %>')">
```

Comment:
need to get the image_path, can't do that in the CSS/less. we won't get a larger version of the image until tomorrow